api/python/notebooks/analysis_demo/comp_bio_census_info.ipynb

```diff
@@ -1310,11 +1310,11 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.6"
+   "version": "3.9.16"
```

Comment:
nit: should we stick a specified python version across all notebooks?

Reply:
Can you elaborate the benefit of doing so?  It seems like we would have to manually enforce this convention (i.e., all of us just remember to use a given version)?

Reply:
Probably not an issue if all notebooks are compliant with the lowest supported Cell Census Python version.

Reply:
There are some protections in place, in `formatting.yml`, which lints against Py 3.7.  This isn't a 100% guarantee, but it helps. If you see more we can do, LMK and I'll get it into an issue